src/collectives/device/onerank_reduce.cu

```diff
@@ -16,7 +16,7 @@ namespace {
     ncclWork *w = &ncclShmem.work;
     int tid = threadIdx.x;
     int tn = blockDim.x;
-    #pragma unroll 1
+    #pragma unroll
```

Comment:
Suggest to revert this change. Original unroll 1 is in NCCL. Changing this may have unexpected effect.